package.json

```diff
@@ -69,13 +70,14 @@
     "yup": "^0.26.2"
   },
   "scripts": {
+    "doc:api":"./node_modules/.bin/redoc-cli bundle build/openapi.yaml -o bin/api-doc.html",
     "translations": "node ./build/translations/csv-json.js",
     "translations:sync": "node ./build/translations/csv-sync.js",
     "build": "npm run translations && ./node_modules/.bin/webpack -d",
     "ui": "npm run translations && ./node_modules/.bin/webpack --mode=production",
-    "ui-dev": "npm run translations && ./node_modules/.bin/webpack -d --watch",
-    "js-lint": "./node_modules/.bin/standard  front-end/src/**/*",
```

Comment:
@zekth this is correct?

Reply:
js-lint was not used anywhere. But after seeing this. Deleting ui-dev is wrong. I'll fix this. Sorry

Reply:
Totally wrong... Don't remember why i've done that. Will rollback sorry.